AppleALC/ALCUserClient/ALCUserClient.hpp

```diff
@@ -0,0 +1,44 @@
+//
+//  ALCUserClient.hpp
+//  AppleALC
+//
+//  Created by Nick on 10/14/20.
+//  Copyright © 2020 vit9696. All rights reserved.
+//
+
+#ifndef ALCUserClient_hpp
+#define ALCUserClient_hpp
+
+#include <IOKit/IOService.h>
+#include <IOKit/IOUserClient.h>
+
+#include "../UserKernelShared.h"
+#include "../ALCUserClientProvider/ALCUserClientProvider.hpp"
+
+class ALCUserClient : public IOUserClient {
+	typedef IOUserClient super;
+	OSDeclareDefaultStructors(ALCUserClient);
+	
+private:
+	ALCUserClientProvider* mProvider;
```

Comment:
Better just null them inline via in-place initialisers like {nullptr};

Reply:
Done.